Changelog.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+### 1.0.0-beta.3
+
+This release only supports Kubernetes versions >= v1.19. The support for Ingress Object in `networking.k8s.io/v1beta` is being dropped and manifests should now use `networking.k8s.io/v1`.
+
+**Image:**
+
+- `k8s.gcr.io/ingress-nginx/controller:v1.0.0-beta.3@sha256:44a7a06b71187a4529b0a9edee5cc22bdf71b414470eff696c3869ea8d90a695`
+
+Changes:
+
+- [X] [#7487](https://github.com/kubernetes/ingress-nginx/pull/7487)[Cherry - Pick] - Fix default backend annotation and tests (#7487)
```

Comment:
I think we need to add 
Add controller.watchIngressWithoutClass config option (#7459) 
Release new helm chart with certgen fixed (#7478)

Reply:
hmm yes. stupid me .. beta.2 had certgen fixed and watcing without ingressClass I will add now after checking older changelog